rmgpy/molecule/translatorTest.py

```diff
@@ -341,29 +335,28 @@ def test_singlet_vs_closed_shell(self):
     #         aug_inchi = 'InChI=1S/C6H5/c1-2-4-6-5-3-1/h1-5H/u1'
     #         self.compare(adjlist, aug_inchi)
 
-    def test_C5H6_triplet_singlet(self):
+    def test_C5H6_singlet(self):
         """
-        n-C5 chain with 2 unpaired electrons at the terminal carbon atoms,
-        and 2 carbon atoms with each a lone pair, next to a terminal
-        carbon atom.
+        n-C5 chain with 1 lone pair at the central carbon atom
```

Comment:
Yes, I changed some tests here because they were testing non-representative structures, and when generating resonance structures in `compare()`, the desired ones got filtered out. I figured we might as well test the properties we're interested in using representative structures.